src/main/java/leaf/cosmere/common/eventHandlers/EntityEventHandler.java

```diff
@@ -159,8 +160,18 @@ public static void giveEntityStartingManifestation(LivingEntity entity, Spiritwe
 				//at this time, players are twin-born minimum, so no need to try give powers here
 			}
 		}
+
+		for (Manifestation manifestation : CosmereAPI.manifestationRegistry())
```

Comment:
Not sure I'm following what you're saying here

Reply:
You're treating each thing you can do with sand mastery as a different power, rather than ways to use a single power. 
You could instead use a single attribute that all your sandmastery powers reference, by overriding getAttribute()

This is how I'm going to get atium to be used by people who have access to electrum.
![image](https://user-images.githubusercontent.com/80794784/203145703-5fa57df6-5695-4c6f-96e5-9bec56265534.png)